docs/interpolation.md

```
@@ -0,0 +1,235 @@
+# Four interior waypoints → 5 equal segments
+pts = interpolate((0.0, 0.0), (0.0, 10.0), n_points=4)
+for i, p in enumerate(pts, 1):
+    print(f"  Waypoint {i}: ({p[0]:.4f}, {p[1]:.4f})")
+# → ~2°, ~4°, ~6°, ~8° longitude
+```
+
+### Understanding the Output
+
+For `n_points=N`, the geodesic is divided into N+1 segments:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)